ketcher-autotests/tests/User-Interface/Hot-Keys/hotkeys.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, test } from '@playwright/test';
+import {
+  selectNestedTool,
+  SelectTool,
+  selectTool,
+  LeftPanelButton,
+  clickInTheMiddleOfTheScreen,
+  waitForPageInit,
+} from '@utils';
+
+test.describe('Hot keys', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('select last chosen selected tool when user press ESC', async ({
+    page,
+  }) => {
+    await selectNestedTool(page, SelectTool.FRAGMENT_SELECTION);
+    await selectTool(LeftPanelButton.AddText, page);
+    await page.keyboard.press('Escape');
+    expect(page.getByTestId('select-fragment')).toBeVisible();
```

Comment:
I see these two tests are failing. Could you, please, add `await` for expect? https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible